sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/src/OpenTelemetryBuilderExtensions.cs

```diff
@@ -184,6 +184,19 @@ public static OpenTelemetryBuilder UseAzureMonitor(this OpenTelemetryBuilder bui
                         azureMonitorOptions.Get(Options.DefaultName).SetValueToLiveMetricsExporterOptions(exporterOptions);
                     });
 
+            builder.Services.AddHostedService(sp =>
+            {
+                Type logForwarderType = Type.GetType("Microsoft.Extensions.Azure.AzureEventSourceLogForwarder, Microsoft.Extensions.Azure", false);
+
+                if (logForwarderType == null || sp.GetService(logForwarderType) == null)
+                {
+                    var loggerFactory = sp.GetRequiredService<ILoggerFactory>();
+                    return new AzureEventSourceLogForwarder(loggerFactory);
+                }
+
+                return AzureEventSourceLogForwarder.Noop;
```

Comment:
Should at some EventSource log here with Noop to indicate that we failed to add the AzureEventSourceLogForwarder because...